apps/api/v2/src/ee/bookings/2024-08-13/services/output.service.ts

```diff
@@ -0,0 +1,128 @@
+
+@Injectable()
+export class OutputBookingsService_2024_08_13 {
+  constructor(private readonly bookingsRepository: BookingsRepository_2024_08_13) {}
+
+  getOutputBooking(databaseBooking: DatabaseBooking) {
+    const dateStart = DateTime.fromISO(databaseBooking.startTime.toISOString());
+    const dateEnd = DateTime.fromISO(databaseBooking.endTime.toISOString());
+    const duration = dateEnd.diff(dateStart, "minutes").minutes;
+
+    const bookingResponses = bookingResponsesSchema.parse(databaseBooking.responses);
+    const attendee = databaseBooking.attendees.find((attendee) => attendee.email === bookingResponses.email);
+
+    if (!attendee) {
+      throw new Error("Attendee not found");
+    }
+
+    const booking = {
+      id: databaseBooking.id,
+      uid: databaseBooking.uid,
+      hosts: [databaseBooking.user],
+      status: databaseBooking.status.toLowerCase(),
+      cancellationReason: databaseBooking.cancellationReason || undefined,
+      reschedulingReason: bookingResponses?.rescheduledReason,
+      rescheduledFromUid: databaseBooking.fromReschedule || undefined,
+      start: databaseBooking.startTime,
+      end: databaseBooking.endTime,
+      duration,
+      eventTypeId: databaseBooking.eventTypeId,
+      attendees: databaseBooking.attendees.map((attendee) => ({
+        name: attendee.name,
+        email: attendee.email,
+        timeZone: attendee.timeZone,
+        language: attendee.locale,
+        absent: !!attendee.noShow,
+      })),
+      guests: bookingResponses.guests,
+      meetingUrl: databaseBooking.location,
+      absentHost: !!databaseBooking.noShowHost,
+    };
+
+    return plainToClass(BookingOutput_2024_08_13, booking, { strategy: "excludeAll" });
+  }
+
+  async getOutputRecurringBookings(databaseBookings: DatabaseBooking[]) {
+    const transformed = [];
+
+    for (const booking of databaseBookings) {
+      if (!booking.id) {
+        throw new Error("Booking was not created");
+      }
+
+      const databaseBooking = await this.bookingsRepository.getByIdWithAttendeesAndUser(booking.id);
+      if (!databaseBooking) {
+        throw new Error(`Booking with id=${booking.id} was not found in the database`);
+      }
+
+      transformed.push(this.getOutputRecurringBooking(databaseBooking));
+    }
+
+    return transformed.sort((a, b) => new Date(a.start).getTime() - new Date(b.start).getTime());
+  }
+
+  getOutputRecurringBooking(databaseBooking: DatabaseBooking) {
+    const dateStart = DateTime.fromISO(databaseBooking.startTime.toISOString());
+    const dateEnd = DateTime.fromISO(databaseBooking.endTime.toISOString());
+    const duration = dateEnd.diff(dateStart, "minutes").minutes;
+
+    const bookingResponses = bookingResponsesSchema.parse(databaseBooking.responses);
+    const attendee = databaseBooking.attendees.find((attendee) => attendee.email === bookingResponses.email);
+
+    if (!attendee) {
+      throw new Error("Attendee not found");
+    }
```

Comment:
maybe add the email of the attendee not found

Reply:
Was unused variable and removed it in [refactor: remove unused attendee variable](https://github.com/calcom/cal.com/pull/16200/commits/441ac833ae9720a31e3af5a09d34fe6c97c0f042)